docs/ledger-nano.md

```diff
@@ -0,0 +1,62 @@
+# Ledger Nano Support
+
+## Prepare Ledger Nano Device to Use with DCL
+
+### Step 1: Install Ledger Live
+
+Download and install Ledger Live: Go to the [Ledger Live download page](https://www.ledger.com/ledger-live) and select the version suitable for your operating system. Follow the on-screen instructions to install it.
+
+<!-- markdown-link-check-disable -->
+### Step 2: Set Up Your Ledger Device
+
+Initialize Your Ledger Device: If this is your first time using your Ledger, you will need to configure it. Follow the instructions to choose a PIN and securely write down the mnemonic seed phrase that appears. This seed is crucial for recovery and is compatible with D∂CL. Detailed instructions are available [here](https://support.ledger.com/hc/en-us/articles/4416927988625-Set-up-your-Ledger-Nano-S-Plus?docs=true).
+
+### Step 3: Install the Cosmos (ATOM) app on your Ledger device
+
+To use the Ledger Nano with DCL, you first need to install the Cosmos (ATOM) application. Connect your Ledger device to your computer, open Ledger Live, and navigate to the 'Manager' tab. Allow the manager on your Ledger device, find the Cosmos (ATOM) app in the catalog, and click 'Install'. For detailed instructions, visit the [Cosmos (ATOM) documentation page](https://support.ledger.com/hc/en-us/articles/360013713840-Cosmos-ATOM?docs=true).
+<!-- markdown-link-check-enable -->
+
+## Using Ledger Device with DCL CLI
+
+To use the DCL (dcld) command line interface, it must first be installed on your computer. You can find the installation instructions in the [Quick start guide](https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/50ef77243b49764f474e545cc4be2beee4793ed0/docs/quickStartGuide.adoc#L3).
```

Comment:
The quick start guide mentions DCL v1.2. I think we need to update the quick start. Probably we should say install the latest version of DCL from https://github.com/zigbee-alliance/distributed-compliance-ledger/releases instead of mentioning the exact version.